src/app/core/facades/shopping.facade.ts

```diff
@@ -109,6 +121,21 @@ export class ShoppingFacade {
     return this.store.pipe(select(getProductListingView(id)));
   }
 
+  productListingViewWithParams$(categoryId: string, productFilter?: string, scope?: string) {
+    const id = ProductListingHelper.getProductListingID(categoryId, productFilter, scope);
+
+    this.store.dispatch(
+      loadMoreProductsForParams({
```

Comment:
As already mentioned `loadMoreProductsForParams` does not seem to be the best, specific enough action to reuse. I guess `page` and `sorting` is not needed while you would need something like `amount` to fetch the desired number of products.